docs/api-docs/storefront/widgets/widget-builder.md

```diff
@@ -0,0 +1,187 @@
+  validate [options] <file>
+  create <widget-template-name>  Create a blank widget template
+  publish <widget-template>      Releases the widget template to the store
+                                 belonging to the env config
+  help [command]                 display help for command
+```
+
+## Configure Widget Builder
+
+Once you have installed Widget Builder, the next step is to set up the widget development environment for your store.  This will authorize you to access your store's data during the development process. You will need the store's `Client ID`, `Access Token`, and `API Path` to generate the appropriate configuration. 
+
+To get started, locate or create a parent directory for the widgets you plan to develop using these credentials.
+
+**Sample input:**
+
+```shell
+# organized by name of store  
+cd ~/bc-projects/tools-r-us
+mkdir widget-workshop
+cd widget-workshop
+```
+
+Next, start the interactive sequence that will generate a `.env` file to authenticate the widgets you build in the subfolders of this parent directory.  The session will prompt you to enter your API account credentials. Begin by running `widget-builder init`. 
+
+**Sample session:**
+
+```shell
+~/bc-projects/tools-r-us/widget-workshop % widget-builder init
+
+Thank you for using Widget Builder
+
+            
+This guide will help you get your environment set up.
+
+Before continuing, please make sure you've created or received a Store API account.
+You'll need those credentials in order to generate the appropriate configurations.
+You can find more information here. https://support.bigcommerce.com/s/article/Store-API-Accounts#creating
+
+? Are you ready to continue? You may press any key to continue (Y/n) Yes
+? What is the Client ID? xxxxxxxxxxxxx
+? What is the Access Token? xxxxxxxxxxxxx
+? What is the API Path? https://api.bigcommerce.com/stores/xxxxx/v3/
+[2021-09-08T15:12:40.271Z] Successfully created your configuration, you're all set!
+```
+## Reset or create parallel configurations
+
+If you need to reset the configuration, you can run the `widget-builder init` command in the same directory to overwrite the existing assignments.  Running `widget-builder init` in sibling directory will create a separate development environment.  
+
+Initializing Widget Builder at multiple levels of a single file hierarchy can have unpredictable results and is not recommended.
+ 
+ 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warn">
+<div class="HubBlock-content">
+
+<!-- theme: warn -->
+
+> To use Widget Builder as a development tool, you must run `widget-builder` commmands from the same directory where you ran `widget-builder init`.
+
+</div>
+</div>
+</div>
+  
+## Generate a scaffold with `create`
+
+To jump start widget development, Widget Builder's `create` command creates a child directory, populates it with the blank template files you need to get started, and spins up a development server.  To use the generator, run the following command:
```

Comment:
To jump-start widget development, begin with the Widget Builder's `create` command, which creates a child directory, populates it with the blank template files you need to get started, and spins up a development server. To use the generator, run the following command:

Reply:
Revisions in [commit 2bc1188](https://github.com/bigcommerce/dev-docs/pull/1200/commits/2bc1188132148dfdbf8190f63b162a853636ac7b) address these concerns.